ansible_collections/arista/avd/roles/eos_designs/docs/tables/svi-profiles.md

```diff
@@ -296,6 +306,12 @@
             # IPv6_address
           - <str>
 
+        # Access-group is assigned on an interface to filter the data packets as they enter the interface.
+        access_group_in: <str>
```

Comment:
access-group reflecting two time, is this expected or we are missing something?

Reply:
One is for Inbound and other is for Outbound.